agent/statemanager/state_manager_test.go

```diff
@@ -128,3 +128,91 @@ func TestLoadsV13DataCorrectly(t *testing.T) {
 	assert.Equal(t, deadTask.GetKnownStatusTime(), expected)
 
 }
+
+// verify that the state manager correctly loads container health check related fields in state file
+func TestLoadsDataForContainerHealthCheckTask(t *testing.T) {
+	cleanup, err := setupWindowsTest(filepath.Join(".", "testdata", "v10", "container-health-check", "ecs_agent_data.json"))
```

Comment:
Should this be in a windows test file?

Reply:
tests in this file will be run on both windows and linux, and this function is doing something that's additionally required on windows, so i think it needs to be here..